src/revivalpmmp/pureentities/PureEntities.php

```diff
@@ -86,21 +85,20 @@ class PureEntities extends PluginBase implements CommandExecutor {
     private static $instance;
 
     /** @var string $loglevel */
-    private static $loglevel;
-    /** @var CustomLogger $logger */
-    private static $logger;
+    private static $loglevel; // please don't change back to int - makes no sense - string is more human readable
```

Comment:
Don't modify anything to do with the logger

Reply:
This and several other changes have resulted from a merge with master, and looking at the gitlog branching for this repo, something has gone wrong. I didn't touch logging at all (for example this comment and lines 91-93 just below were merged from master, and I didn't add "STAND" either) . These changes resulted from a merge but hasn't been recognised as such by git - and since I can't find the blame for this and other changes here in the commit history, I suspect that the recent master merge and reverted is to blame. Maybe @95CivicSi has an idea what happened?

Reply:
The logger in PureEntities was made async by myself if that's what you mean